corehq/apps/app_manager/templatetags/xforms_extras.py

```diff
@@ -29,7 +29,7 @@ def trans(name, langs=None, include_lang=True, use_delim=True, prefix=False, esc
     else:
         tag = lambda lang: ""
     for lang in langs:
-        if lang in name and name[lang]:
+        if lang in name and name[lang] is not None:
```

Comment:
This one change is not related to the feature but had to do this here. Details in [commit](https://github.com/dimagi/commcare-hq/pull/16871/commits/14453f4482f6580adc9619a8ad3efb39d5cf37a2)